demo/api/src/main.ts

```diff
@@ -39,6 +41,11 @@ async function bootstrap(): Promise<void> {
         }),
     );
 
+    app.use(
+        helmet({
+            contentSecurityPolicy: false, // configure this when API returns HTML
+        }),
+    );
```

Comment:
Is already present in the starter and causes the PDF bug (https://github.com/vivid-planet/comet-starter/blob/main/api/src/main.ts#L48-L52)